.github/workflows/deploy-unix.yml

```diff
@@ -21,11 +21,21 @@ jobs:
             TARGET: aarch64-unknown-linux-gnu
             SUFFIX: aarch64
 
+          - os: macos-13-xlarge
+            TARGET: aarch64-apple-darwin
+            SUFFIX: macos-aarch64
+
+          - os: windows-latest
```

Comment:
```suggestion
          - os: windows-2022
```

Reply:
`latest` is always out of out control, then it updates and the build suddenly starts breaking